src/main/webapp/index.html

```diff
@@ -36,8 +36,8 @@
       <div class="form-group">
         <nav>
           <div class="nav nav-tabs" id="nav-tab" role="tablist">
-            <a class="nav-item nav-link active" id="nav-keyword-tab" data-toggle="tab" href="#nav-keyword" role="tab" aria-controls="nav-keyword" aria-selected="true">Keyword Search</a>
-            <a class="nav-item nav-link" id="nav-videolink-tab" data-toggle="tab" href="#nav-videolink" role="tab" aria-controls="nav-videolink" aria-selected="false">Video Link</a>
+            <a class="nav-item nav-link active" onClick="hideLinkInfo()" id="nav-keyword-tab" data-toggle="tab" href="#nav-keyword" role="tab" aria-controls="nav-keyword" aria-selected="true">Keyword Search</a>
+            <a class="nav-item nav-link" onClick="hideSearchInfo()" id="nav-videolink-tab" data-toggle="tab" href="#nav-videolink" role="tab" aria-controls="nav-videolink" aria-selected="false">Video Link</a>
```

Comment:
cannot imagine that this is working...

Reply:
hahaha it's so much simpler than we were looking in to